src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -302,7 +302,7 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 	matrix::Vector3f _body_velocity{};
 
 	bool _reinitialize_tecs{true};
-	bool _is_tecs_running{false};
+	bool _tecs_is_running{false};
```

Comment:
Super minor, but it might be nice to eliminate extra state like this entirely by simply calling into `_tecs`.